ecogvis/functions/misc_dialogs.py

```diff
@@ -1541,9 +1598,1119 @@ def paint(self):
         self.GLViewWidget.renderText(self.X, self.Y, self.Z, self.text)
 
 
-# Gray line for visual separation of buttons -----------------------------------
+# Creates Event-Related Potential dialog ------------------------------------
+class ERPDialog(QMainWindow):
```

Comment:
this shouldn't be here, this class has been moved to its own module 'event_related_potential.py'

Reply:
ignore that, we can move it later, the most important is to keep their changes